tests/device_tests/cardano/test_address_public_key.py

```diff
@@ -0,0 +1,82 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib.cardano import create_address_parameters, get_address, get_public_key
+from trezorlib.messages import CardanoAddressType
+from trezorlib.tools import parse_path
+
+from ...common import parametrize_using_common_fixtures
+
+pytestmark = [
+    pytest.mark.altcoin,
+    pytest.mark.cardano,
+    pytest.mark.skip_t1,
+    pytest.mark.skip_ui,
+]
+
+ADDRESS_TYPES = {
```

Comment:
please also remove this

Reply:
Aaah. Done and I have fixed style, which makes the `.. if .. else ..` really ugly.. Maybe I should revert that to the first thing?

Reply:
I don't mind this formatting actually, but feel free to revert to the original version if you dislike it. I don't think either version better than the other.